macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql

```diff
@@ -32,7 +31,9 @@
                 {% if dimensions %}
```

Comment:
I wonder if the `dimensions` should also be part of the `metric_properties`.
What do you think?

Reply:
I would hate to increase the scope of this feature, TBH. About the actual question, I don't know enough about the usage / meaning of `dimensions` to provide an answer